app/components/sidebars/main/channels_list/channel_item/channel_item.js

```diff
@@ -135,6 +135,7 @@ export default class ChannelItem extends PureComponent {
 
         let badge;
         if (mentions) {
+            const maxWidth = mentions > 99 ? 38 : 32;
```

Comment:
maybe this should be handled by the layout and not define a maxWidth as based on previous feedback this could be causing the text to be cutoff